language/tools/move-mv-llvm-compiler/src/stackless/llvm.rs

```diff
@@ -444,6 +468,38 @@ impl Builder {
         }
     }
 
+    /// Get a struct element.
```

Comment:
This is almost a copy of `field_ref()` (but takes `AnyValue` which is a bit more general). The name is also better for us LLVM types. So perhaps just remove `field_ref` altogether (it isn't used right now).